hack/BUILD.bazel

```diff
@@ -183,6 +185,17 @@ sh_test(
     ],
 )
 
+sh_binary(
+    name = "update-golangci-lint",
```

Comment:
/nit since we're not updating anything, I think we should remove the `update-` prefix here (for the script as well).

Reply:
Removed prefix